src/query-builder.ts

```diff
@@ -1,18 +1,32 @@
 import { TaggedTypeFormat } from "./tagged-type";
 import type {
-  QueryValueObject,
+  FQLFragment,
   QueryValue,
   QueryInterpolation,
-  QueryRequest,
-  QueryOptions,
 } from "./wire-protocol";
 
+/**
+ * A QueryArgumentObject is a plain javascript object where each property is a
+ * valid QueryArgument.
+ * These objects can be set as values in the {@link fql} function.
+ */
+export type QueryArgumentObject = {
+  [key: string]: QueryArgument;
+};
+
+/**
+ * A QueryArgument is a plain javascript object where each property is a
+ * valid QueryArgument.
```

Comment:
I don't think this comment is correct

Reply:
Thanks for catching the copy-pasta! I've updated in [7bb285a](https://github.com/fauna/fauna-js/pull/272/commits/7bb285a318d9096ded589200a4e2ee7bab325e81). How's that?